### DISCUSSION TITLE
Release 3.5.2

### DISCUSSION THREAD

**Discussion by sofoklis** [slick/slick#2981] [Questions] [answered]
When is the plan to release the next version?

**Comment by nafg** [comment]
Hi, sorry for not responding earlier.

I think since we're currently doing PVP the current changes since 3.5.1 should indeed be released as 3.5.2, but sbt-version-policy complains that we bumped HikariCP from 4 to 5 which implies breaking binary compatibility. I don't think that's the case, so I opened https://github.com/slick/slick/pull/2984, but I'm not sure what the "right" thing to do is.

Thoughts?

**Reply by nafg** [reply]
Maybe in any case I should bump to 3.6.0 simply because we too have dropped Java 8.

**Reply by sofoklis** [reply]
Thanks for the reply, for us version number is not important, either option is ok.

**Comment by nafg** [comment, marked as answer]
Sorry for not following up sooner. I was going to release 3.6.0-M1 and then I got swamped. But somehow I seem to have accidentally tagged 3.5.2 yesterday. So enjoy.

It requires Java 11+. I hope that doesn't cause issues for anyone.
